qiskit_experiments/library/hamiltonian/heat_zx.py

```diff
@@ -0,0 +1,220 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+HEAT experiments for ZX Hamiltonian.
+"""
+
+from typing import Tuple, Optional
+
+import numpy as np
+from qiskit.circuit import QuantumCircuit
+from qiskit.providers import Backend
+
+from .heat_base import BatchHeatHelper, HeatElement
+from .heat_analysis import HeatAnalysis
+
+
+class ZXHeat(BatchHeatHelper):
+    """HEAT experiment for the ZX-type entangler.
+
+    # section: overview
+        This experiment is designed to amplify the error contained in the
+        ZX-type generator, a typical Hamiltonian implemented
+        by a cross resonance drive, which is a foundation of the CNOT gate.
+
+        The echo circuit refocuses ZX rotation to identity (II) then applies
+        a pi-pulse along the interrogated error axis. X error and Y error are
+        amplified outward the X-Y plane to draw a ping-pong pattern with
+        state flip by the pi-pulse echo, while Z error is amplified outward
+        the X-Z plane in the same manner.
+        Measurement is projected onto Y-axis in this setup.
+        Because the echoed axis are anti-commute with other Pauli terms,
```

Comment:
You need to define the term `eched axis`.
```suggestion
        Because the echoed axis anti-commute with other Pauli terms,
```